src/common/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback, useState } from 'react';
+import styles from './SideBar.scss';
+import SideBarHideSvg from '../../assets/icons/sideBarHide.svg';
+import cn from 'classnames';
+
+interface SideBarProps {
+  title: string;
+  data: [];
+}
+
+export function SideBar({data, title}: SideBarProps) {
+  const [expandedId, setExpadedId] = useState(-1);
+  const changeExpandedIdCb = useCallback((newId:number) => {
+    let id = newId;
+    
+    if (expandedId === id) id = -1;
+    setExpadedId(id);
+  }, [expandedId]);
+
+  return (
+    <div className={styles.sideBar}>
+      <SideBarHeader title={title} onHide={changeExpandedIdCb}/>
```

Comment:
`changeExpandedIdCb` isn’t hide handler.
() => changeExpandedIdCb(-1) - it’s hide handler
You can also rename `onHide` to `onChange`